trestle/core/ssp_io.py

```diff
@@ -213,14 +215,53 @@ def get_control_response(self, control_id: str, level: int) -> str:
         'The System' is the default response, and all other components are treated as sub-headings per response item.
         """
         if not self._profile_path:
-            raise TrestleError('Cannot get control statement, set profile first.')
+            raise TrestleError('Cannot get control response, set profile first.')
 
-        writer = ControlIOWriter()
         control = self._catalog_interface.get_control(control_id)
-        group_id, group_title, _ = self._catalog_interface.get_group_info_by_control(control.id)
-        response_lines = writer.get_response(control)
+        control_impl_req = self._control_implemented_req(control_id)
 
-        tree = MarkdownNode.build_tree_from_markdown(response_lines)
+        md_writer = MDWriter(None)
+        if control_impl_req.statements:
+            for statement in control_impl_req.statements:
+                statement_id = statement.statement_id
+                label = statement_id
+                # look up label for this statement
+                if control.parts:
+                    found_label = self._catalog_interface.get_statement_label_if_exists(control_id, statement_id)
+                    if found_label:
+                        label = found_label
+
+                header = f'Implementation {label}'
```

Comment:
So I think there are some extra checks needed here. If there is only one implemented requirement and no 'parts'  
then it should not have an implementation header. This is analgous to if the statement id corresponds to the top level statement for a control. 

If there is more than one part (and / or at multiple levels) - then you print the implementation ID.

Reply:
Thanks, fixed

Reply:
Now the `Part {label}` will only be printed if the part has the name `item`. If part name is `statement` then only `description` per component in `by-component` will be printed if any